packages/stack/src/components-page/error-page.tsx

```diff
@@ -61,5 +64,22 @@ export function ErrorPage(props: { fullPage?: boolean, searchParams: Record<stri
     );
   }
 
+  if (error instanceof KnownErrors.OAuthProviderAccessDenied) {
```

Comment:
I think instead of rendering this card, we should just redirect back to sign-in or sign-up (wherever we came from)

Reply:
Not a big fan of that UX, I think the user should still see that it failed